modules/monitoring-creating-alert-routing-for-user-defined-projects.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// * monitoring/managing-alerts.adoc
+
+[id="creating-alert-routing-for-user-defined-projects_{context}"]
+= Creating alert routing for user-defined projects
+ 
+[IMPORTANT]
+====
+[subs="attributes+"]
+Alert routing for user-defined projects is a Technology Preview feature only. Technology Preview features are not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete. Red Hat does not recommend using them in production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
```

Comment:
need @PhilipGough help to confirm if the feature is a Technology Preview feature

Reply:
Can confirm this feature is tech preview in 4.10